src/RealtimeClient.ts

```diff
@@ -61,9 +61,15 @@ interface WebSocketLikeError {
 
 const NATIVE_WEBSOCKET_AVAILABLE = typeof WebSocket !== 'undefined'
 
-const WebSocketVariant: WebSocketLikeConstructor = NATIVE_WEBSOCKET_AVAILABLE
-  ? WebSocket
-  : require('ws')
+let WebSocketVariant: WebSocketLikeConstructor
+
+if (NATIVE_WEBSOCKET_AVAILABLE) {
+  WebSocketVariant = WebSocket
+} else {
+  import('ws').then(({ default: WSWebSocket }) => {
```

Comment:
I think this line would fixe the issue, but I think it creates a race condition with `RealtimeClient`, which needs to read it.
Maybe throw an error in `RealtimeClient` if `WebSocetVarient` is not loaded?

Reply:
thanks, we were afraid of this approach introducing the race condition as well.